data/blog/posts/2022/2022-02-07-success-story-of-automated-framework-migration-from-fuelphp-to-laravel-of-400k-lines-application.md

```diff
@@ -178,53 +178,7 @@ public function refactor(Node $selectArrayNode): ?Node
 
 Great! Now we can convert the whole query running these 2 rules.
 
-<br>
-
-Rector can perform much more flexible refactoring. For example, sometimes, the array arg of the `select_array` method is passed by variables.
-
-```php
-select_array($array)
-```
-
-Then we can convert them:
-
-```php
-select(...$array)
-```
-
-<br>
-
-We'll add a little code to handle that case.
-
-```php
-public function refactor(Node $selectArrayNode): ?Node
-{
-    if (!$this->isName($selectArrayNode->name, 'select_array')) {
-        return null;
-    }
-
-    if (count($selectArrayNode->args) !== 1) {
-        return null;
-    }
-
-    $value = $selectArrayNode->args[0]->value;
-    if ($value instanceof Node\Expr\Array_) {
-        $selectArrayNode->args = array_map(
-            fn(Node\Expr\ArrayItem $item) => new Node\Arg($item->value),
-            $value->items
-        );
-    } elseif ($value instanceof Node\Expr\Variable) {
-        $selectArrayNode->args[0]->unpack = true;
-    } else {
-        return null;
-    }
-
-    $selectArrayNode->name = new Node\Identifier('select');
-    return $selectArrayNode;
-}
-```
-
-## Developing new Features and Running Migration at the Same Time
```

Comment:
@rajyan Btw, I've dropped this section, as it seems there are enough Rector rules above. It might be quite tedious to read AST code, unless reader is already familiar with. I think the 2 rules above describe the rule functionality well enough :+1: 

Is it ok for you?

Reply:
I agree with you :+1: 
Let's delete it.